gruntfile.js

```diff
@@ -3,30 +3,53 @@ module.exports = function(grunt) {
   // Project configuration.
   grunt.initConfig({
     pkg: grunt.file.readJSON('package.json'),
+
     sass: {
-      options: {
+      option: {
```

Comment:
missing `s`
and according to (the doc)[https://github.com/gruntjs/grunt-contrib-sass#example-config] `options` should be in `dist`